docs/querying/granularities.md

```diff
@@ -41,7 +41,7 @@ Simple granularities are specified as a string and bucket timestamps by their UT
 Supported granularity strings are: `all`, `none`, `second`, `minute`, `fifteen_minute`, `thirty_minute`, `hour`, `day`, `week`, `month`, `quarter` and `year`.
 
 * `all` buckets everything into a single bucket
-* `none` does not bucket data (it actually uses the granularity of the index - minimum here is `none` which means millisecond granularity). Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended (the system will try to generate 0 values for all milliseconds that didn’t exist, which is often a lot).
+* `none` actually does bucket data - to the granularity of the internal index - which means millisecond granularity. `none` can be thought of as `millisecond`.  Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended (the system will try to generate 0 values for all milliseconds that didn’t exist, which is often a lot).
```

Comment:
```suggestion
* `none` buckets data to the granularity of the internal index which is millisecond granularity. You can think of `none` as equivalent to `millisecond`.  Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended because the system will try to generate 0 values for all milliseconds that didn’t exist, which can be a lot.
```